IoTuring/Configurator/Configurator.py

```diff
@@ -170,9 +166,32 @@ def ManageWarehouses(self) -> None:
         if choice == CHOICE_GO_BACK:
             self.Menu()
         else:
-            self.ManageSingleWarehouse(choice, wcm)
+            self.ManageSingleWarehouse(choice)
+
+    def ManageSettings(self) -> None:
+        """ UI for App and Log Settings """
+        choices = [
+            {"name": "Log Settings", "value": Logger},

```

Comment:
@richibrics 
These two classes are hardcoded here and in `ConfiguratorLoader`. Maybe a really basic `SettingsClassManager` could be written for them? I don't know.

Reply:
I see, the problem is that the Manager should look for those classes in the entire project directory if we make an automatic search of those classes (the one for the entities has only to look at the entities deployments).
Otherwise we could hardcode in the new manager the classes, but I don't know

Reply:
Otherwise (I don't know if it's really doable) the classes files may add their classes to a list somewhere that is explored from the configurator when the settings page is opened. For example when the file of the class loads, the append to a list is done.
But that would need the classes files to be imported at start which I think is again a problem (and overall not a great job).

Reply:
The folders of entities and warehouses are already hardcoded in `ClassManager`, we just search in these hardcoded folders. Similarly in this new `SettingsClassManager` the path of the files would be hardcoded, not the folders, so I think it's somewhat similar flow, but on a different level. 

`Entities` and `Warehouses` are loaded in `Configurator` from their `ClassManager`, `Settings` would work the same way. On normal run `ClassManager` is called from `ConfiguratorLoader` and it would work the same way for `Settings`. 

With some changes in `ClassManager`s all methods could be the same for all 3, currently there some methods not called from anywhere.

I can rewrite `ClassManager` as well, should I just commit here? Or if you pull this, I can start a new PR for that, as it's not necessarily related to this PR.

Reply:
Thought about it, I think a new SettingsClassManager could be the way to go. For the rewriting of ClassManager I let you choose since currently I can't work on it

Reply:
I already started working on it, it looks good but buggy, I haven't pushed it yet.